test/videojs.ima.test.js

```diff
@@ -58,3 +58,45 @@ test('ad plays on good ad tag', function() {
     start();
   }, 5000);
 });
+
+test('video continues after ad was skipped', function() {
+  var options = {
+    id: 'video',
+    adTagUrl: 'http://pubads.g.doubleclick.net/gampad/ads?sz=640x360&' +
+        'iu=/6062/iab_vast_samples/skippable&ciu_szs=300x250,728x90&impl=s&' +
+        'gdfp_req=1&env=vp&output=xml_vast2&unviewed_position_start=1&' +
+        'url=[referrer_url]&correlator=[timestamp]'
+  }
+
+  //addEventListener only works when the adManager is available, thus using it in the ready-callback
+  var readyForPrerollCallback = function() {
+    player.ima.addEventListener(google.ima.AdEvent.Type.SKIPPABLE_STATE_CHANGED, function() {
+      //https://goo.gl/fgoJU1
```

Comment:
Could you change this comment to just indicate that this is AdsManager.skip() without the shortlink? We try to avoid shortlinks in code comments because you don't know what could be on the other end.